Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -62,12 +62,15 @@ public ShooterWheelSubsystem(int motor, RobotSide side, WPIFactory factory, XPro
         shooterTalonError = propManager.createEphemeralProperty(side + "Shooter speed error", 0);
         enableShooterLogging = propManager.createEphemeralProperty("Is " + side + "Shooter logging info?", false);
         
-        this.masterMotor = factory.getCANTalonSpeedController(motor);
-        setUpMotorTeam(masterMotor);
-        masterMotor.createTelemetryProperties(side + "Shooter master", propManager);
+        this.masterMotor = factory.getCANTalonSpeedController(masterChannel);
+        this.followerMotor = factory.getCANTalonSpeedController(followerChannel);
+        
+        setUpMotorTeam(masterMotor, followerMotor);
+        masterMotor.createTelemetryProperties(side + "Shooter wheel master", propManager);
```

Comment:
Add a space in the string between the number and "shooter" (also, decapitalize "Shooter")